src/software/field_tests/field_test_fixture.py

```diff
@@ -320,6 +320,27 @@ def load_command_line_arguments():
         help="Run the test with friendly robots in yellow mode",
     )
 
+    parser.add_argument(
+        "--sslvision_multicast_address",
+        type=str,
+        default=SSL_VISION_ADDRESS,
+        help="the multicast address for ssl vision",
+    )
+
+    parser.add_argument(
+        "--use_unconventional_port",
+        action="store_true",
+        default=False,
+        help="setting this option would cause gamecontroller to bind on an unconventional port, likely 12393",
+    )
```

Comment:
we don't need this as @nimazareian suggested. If we run `--launch_gc`, then we can set the appropriate port in `RobotCommunication`